README.md

```diff
@@ -241,6 +241,11 @@ spec:
 
 You can only select from `internet-facing` (default) and `internal` options.
 
+By default the ingress-controller will aggregate all ingresses under a single
```

Comment:
single? if you have more than 25 you have 2 :)

Reply:
changed it to "as few as possible" :)